docs/ui/components/Footer/NewsletterSignUp.tsx

```diff
@@ -59,19 +50,22 @@ export const NewsletterSignUp = () => {
           event.preventDefault();
           signUp();
         }}>
-        <Input
-          onChange={event => {
-            setEmail(event.target.value);
-            if (event.target.value.length === 0) {
-              setError(null);
-            }
-          }}
-          value={email}
-          className={mergeClasses('pr-[68px]', error && 'border-danger text-danger')}
-          type="email"
-          placeholder={userSignedUp ? 'Thank you for the sign up!' : 'your@email.com'}
-          disabled={userSignedUp}
-        />
+        {userSignedUp ? (
+          <LABEL className="flex items-center my-2.5 h-12">Thank you for the sign up! 💙</LABEL>
+        ) : (
+          <Input
+            onChange={event => {
+              setEmail(event.target.value);
+              if (event.target.value.length === 0) {
+                setError(null);
+              }
+            }}
+            value={email}
+            className={mergeClasses('pr-[68px]', error && 'border-danger text-danger')}
+            type="email"
+            placeholder="your@email.com"
```

Comment:
```suggestion
            placeholder="reader@example.com"
```

Let's update this since this is what we use on our mailing list page on the website.

![CleanShot 2023-12-13 at 00 57 03@2x](https://github.com/expo/expo/assets/10234615/67f7060d-279a-4cef-bde8-59e2749025b0)

Reply:
Good idea! 👍